test/arrays/bradc/formalActualArrayArgMatch.chpl

```diff
@@ -0,0 +1,45 @@
+use BlockDist;
```

Comment:
I think I actually need a skipif given that some of the compopts work in either case and some don't (and I don't think we lose much my not testing this under fast) -- sound right?